src/python/ensembl/genes/tracking/bioproject_tracking.py

```diff
@@ -0,0 +1,250 @@
+            if only_haploid and assembly_info.get('assembly_type') != 'haploid':
+                continue
+            assembly_accession = assembly.get('accession')
+            taxon_id = assembly.get('organism').get('tax_id')
+            if assembly_accession:
+                assembly_accessions[assembly_accession] = {"taxon_id" : taxon_id}
+
+        # Check for the next page token or equivalent
+        next_page_token = data.get('next_page_token')
+        if not next_page_token:
+            break
+
+    return assembly_accessions
+
+def get_ensembl_live(bioproject_accessions_taxon):
+    """
+    Retrieves live Ensembl database names for a set of bioproject accessions.
+
+    This function constructs and executes a SQL query to fetch the latest Ensembl database names (denoted as 'dbname')
+    associated with a list of assembly accessions. The query selects databases where the assembly accession matches
+    those provided and filters for 'core' databases in the latest data release. The information is then used to
+    update and return a dictionary mapping each accession to its corresponding taxonomy information and live Ensembl 
+    'dbname'.
+
+    Args:
+        bioproject_accessions_taxon (dict): A dictionary where keys are assembly accessions and values are dictionaries 
+                                            containing taxonomy information for those accessions.
+
+    Returns:
+        dict: A dictionary where each key is an assembly accession, and each value is a dictionary with the original 
+              taxonomy information plus a 'dbname' key containing the name of the live Ensembl database for that accession.
+
+    Requires:
+        - A working connection to the Ensembl metadata database specified by the configuration in `config`.
+        - The `mysql_fetch_data` function to execute the query and fetch data.
+
+    Example:
+        >>> bioproject_accessions_taxon = {'GCA_123456.1': {'taxon_id': 9606}}
+        >>> live_annotations = get_ensembl_live(bioproject_accessions_taxon)
+        >>> print(live_annotations)
+        {'GCA_123456.1': {'taxon_id': 9606, 'dbname': 'homo_sapiens_core_104_38'}}
+
+    Note:
+        The configuration for the database connection (`config`) must be defined externally with keys for 
+        'db_host', 'db_port', and 'db_user' within a nested 'server_details'->'meta' structure.
+    """
+    accessions = list(bioproject_accessions_taxon.keys())
+    formatted_accessions = ",".join(f'"{item}"' for item in accessions)
+    data_query = (
+        "SELECT assembly.assembly_accession, dbname FROM assembly JOIN genome USING (assembly_id) JOIN genome_database USING (genome_id) WHERE genome.data_release_id=(SELECT MAX(data_release_id) FROM genome) AND assembly.assembly_accession in (" + formatted_accessions + ") AND dbname like '%core%';"
+    )
+    data_fetch = mysql_fetch_data(
+        data_query,
+        'ensembl_metadata_qrp',
```

Comment:
I had originally done this, then I had some code to use the ncbi_taxonomy db, then I removed that because it was slow. I want to be able to access different dbs on the same server, so I will rethink the structure of this so it suits our developing needs

Reply:
Do you mean querying different dbs on production server? There might be a general function that has server details  and query as input and return the reply as a tuple

Reply:
Exactly, the Production dbs that we will want to access (in the future the beta metadata, but there may also be others) will all be on the meta server. I re-jigged the config and included the db_name in there.